client/cmd/dexc-desktop/metainfo/org.decred.dcrdex.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.decred.dcrdex</id>
+  <launchable type="desktop-id">org.decred.dcrdex.desktop</launchable>
+  <name>DCRDEX</name>
+  <summary>Decentralized cryptocurrency exchange</summary>
+  <metadata_license>MIT</metadata_license>
+  <project_license>BlueOak-1.0.0</project_license>
+  <url type="homepage">https://dex.decred.org</url>
+  <url type="help">https://chat.decred.org/#/room/#dex:decred.org</url>
+  <developer_name>The Decred Developers</developer_name>
+  <description>
+    <p>
+      Non-custodial peer-to-peer exchange where your funds never leave your wallet.  
+      Trade Bitcoin, USDC, Ethereum, Decred, Dogecoin, Zcash and more without KYC 
+      and fees, using atomic swaps technology.
+    </p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/decred/umbrel-app-store/master/decred-dcrdex/images/screenshot_1.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/decred/umbrel-app-store/master/decred-dcrdex/images/screenshot_2.png</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/decred/umbrel-app-store/master/decred-dcrdex/images/screenshot_3.png</image>
```

Comment:
These images are really outdated. Can we host the images in our own repo?

Reply:
Yes, these are meant to be placeholder images only.  We need to take the screenshots whenever appropriate and the URLs can point to the images in repo.